homework1/styles.css

```diff
@@ -0,0 +1,652 @@
+  gap: 6em;
+}
+
+.content--one__left {
+  display: flex;
+  flex: 0.5;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.content--one__left h3 {
+  font-family: "Baloo2-Bold";
+  color: #000000;
+  font-size: 26px;
+  margin-top: 0;
+}
+
+.content--one__left p {
+  font-size: 14px;
+}
+
+.content--one__left p:nth-of-type(2) {
+  font-family: "Baloo2-Medium";
+}
+.content--one__left div {
+  display: flex;
+  flex-direction: column;
+}
+.content--one__left div ul {
+  list-style: "∙";
+  list-style-position: outside;
+  margin-left: 10px;
+}
+.content--one__left div ul li {
+  font-size: 14px;
+  padding-left: 10px;
+}
+
+.content--one__left div li:last-child a {
+  background-color: #59abb0;
+  border-radius: 10px;
+  padding: 5px 30px;
+  margin-left: 20px;
+}
+
+.apply-btn {
+  margin-top: 2em;
+  width: 18em;
+}
+
+.content--one__right {
+  display: flex;
+  flex: 0.5;
+  width: 50%;
+  justify-content: flex-start;
+}
+
+.content--one__right img {
+  width: 100%;
+}
+
+.content--two {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 6em;
+  margin: 4em auto;
+}
+
+.content--two__left {
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.content--two__left img {
+  width: 100%;
+}
+
+.content--two__right {
+  width: 50%;
+  display: flex;
+
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.content--two__right h4 {
+  font-size: 18px;
+  font-family: "Baloo2-Medium";
+  margin: 0;
+}
+
+.content--two__right p {
+  font-family: "Baloo2-Regular";
+  font-size: 14px;
+}
+.content--two__right--btn {
+  width: 170px;
+}
+
+/* Footer */
+
+footer {
+  width: 100%;
+  justify-content: space-between;
+  background-color: #59abb0;
+  color: #ffffff;
+}
+.footer--container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 90%;
+  margin: 0 auto;
+}
+.footer--container div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 137px;
+}
+
+.footer--container div p {
+  margin: 0;
+}
+
+.footer--container ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+}
+
+.footer--container ul li a {
+  color: #ffffff;
+}
+
+.footer--container ul li a:hover {
+  text-decoration: underline;
+}
+
+/* Button */
+
+.btn {
+  background-color: #59abb0;
+  border-radius: 10px;
+  color: #ffffff;
+  text-align: center;
+}
+
+.btn p {
+  margin: 10px;
+}
+
+.btn:hover {
+  background-color: #60dfe8;
+}
+
+/* header */
+
+.header {
+  background-color: #fff;
+  width: 100%;
+  z-index: 3;
+}
+
+.header--content {
+  width: 90%;
+}
+
+.header ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  overflow: hidden;
+  background-color: #fff;
+}
+
+.header li a {
+  display: block;
+  padding: 10px 20px;
+  text-decoration: none;
+  float: left;
+}
+
+.menu li a {
```

Comment:
it's not a good solution to style by tag. Better if u add some class for it. 
Because somewhere in other part of your site u can use some library or same class name .menu that will contains LI  and A ...

Reply:
Yea that’s true… I will use a different name instead of .menu .
Thanks